controller/k8s/watcher.go

```diff
@@ -36,6 +47,19 @@ func (w *watcher) run() error {
 	initialized := make(chan struct{}, 1)
 	defer close(initialized)
 
+	go wait.ExponentialBackoff(wait.Backoff{
+		Duration: durationBetweenBackoff,
```

Comment:
`durationBetweenBackoff` (and similar constants?) are only used in this one place. The code would be clearer if their values were just inlined here.

One of the main reasons to use ExponentialBackoff was so that the initial backoff could be shorter than 1 second, so that if we need to retry (which is super common, apparently) we don't have to wait 1 full second to try again.

In particular, some of these containers depend on other control-plane containers. Thus, it seems like these backoff periods could end up stacking on top of each other such that we might have to wait multiple seconds for the control plane to be fully ready. That's what I'm hoping to avoid, by setting the initial duration to be much less than 1 second.